solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/listProjectsByEtc.test.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import ClientSession from '../../../support/clientSession';
+import { PaabHelper } from '../../../support/complex/paabHelper';
+import Setup from '../../../support/setup';
+import HttpError from '../../../support/utils/HttpError';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+describe('list environment type configs', () => {
+  const setup: Setup = Setup.getSetup();
+  const envTypeId = setup.getSettings().get('envTypeId');
+  const envTypeConfigId = setup.getSettings().get('envTypeConfigId');
+  const paabHelper: PaabHelper = new PaabHelper();
+  let itAdminSession: ClientSession;
+  let paSession: ClientSession;
+  let researcherSession: ClientSession;
+
+  beforeEach(() => {
+    expect.hasAssertions();
+  });
+
+  beforeAll(async () => {
+    const paabResources = await paabHelper.createResources(__filename);
+    itAdminSession = paabResources.adminSession;
+    paSession = paabResources.pa1Session;
+    researcherSession = paabResources.rs1Session;
+  });
+
+  afterAll(async () => {
+    await paabHelper.cleanup();
+    await setup.cleanup();
+  });
+
+  describe('ITAdmin tests', () => {
```

Comment:
Should these tests go into this preexisting file?
https://github.com/aws-solutions/research-service-workbench-on-aws/blob/2645d12a07ae9b9ba185e234713c55639e96291d/solutions/swb-reference/integration-tests/tests/isolated/projectEnvTypeConfigs/listProjects.test.ts